sips/minutes/2016-10-25-sip-minutes.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: sip-landing
+title: SIP Meeting Minutes - 25th October 2016
+---
+
+# Minutes
+
+The following agenda was distributed to attendees:
+
+| Topic | Reviewer |
+| --- | --- |
+| Discussion of the voting system | N/A |
+| [SIP-20: Improved Lazy Val Initialization](http://docs.scala-lang.org/sips/pending/improved-lazy-val-initialization.html) | Sébastien Doeraene |
+| [SIP-27: Trailing commas](https://github.com/scala/scala.github.com/pull/533#issuecomment-232959066) | Eugene Burmako |
+
+Jorge Vicente Cantero was the Process Lead and acting secretary of the meeting.
+
+* **Decided to reaccept for review**: SIP-27: Trailing commas
+* **Under review**: SIP-20: Improved Lazy Val Initialization
+
+## Date, Time and Location
+
+The meeting took place at 5:00pm Central European Time / 8:00am Pacific Daylight
+Time on Tuesday, October 25th, 2016 via Google Hangouts.
+
+Minutes were taken by Jorge Vicente Cantero, acting secretary.
+
+## Attendees
+
+Attendees Present:
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Seth Tisue ([@SethTisue](https://github.com/SethTisue)), EPFL
```

Comment:
Let me explain everything in the submission process, and we discuss the clarity of the rules there :smile:. If necessary, I can come back to these minutes and modify them accordingly or link to the actual changes.